aea/cli/create.py

```diff
@@ -84,6 +85,14 @@ def create(click_context, agent_name):
         )
         sys.exit(1)
 
+    config = _get_or_create_cli_config()
+    author = config.get(AUTHOR, None)
+    if author is None:
+        click.echo(
+            "The AEA configurations are not initialized. Uses `aea init` before continuing."
```

Comment:
We need to interrupt create if the author is not set. Otherwise the author fields are all wrong in the configs. This introduces large changes to the tests!

Reply:
We could make this step optional and stick with the old version. So it is also not backward-incompatible.

Reply:
The problem with this being optional is that it really is not. The current behaviour is buggy as it introduces the `"author"` value as the `author`, which just doesn't make sense and causes for inconsistencies. So really this is a bug fix. The only option I see is to pass in optional arguments `--author` to `aea create [some_agent]`.

Reply:
Agreed. Let's support also `aea create [someagent] --author [author_name], and then update the docs accordingly.

Reply:
The docs can be staying the same. I added an init intro to the Quickstart.